### PR TITLE
feat created a button to mockup design

### PR DESCRIPTION
**Pull request by Sreelakshmi1831**
## Feature description
Need to creat a button to mockup design

## Solution description
Created a button to mockup design from feasibility check .
Only show the button when feasibility is approved.
## Output
image

## Areas affected and ensured
-New feature

## Is there any existing behavior change of other features due to this code change?
-No

## Was this feature tested on the browsers?
  - Windows edge

### CODE REVIEW

File: versa_system/versa_system/doctype/feasibility_check/feasibility_check.js [hunk 1]
```diff
@@ -1,8 +1,23 @@
-// Copyright (c) 2024, efeone and contributors
-// For license information, please see license.txt
-
```
Comment (MhmdSinanKT) [inline versa_system/versa_system/doctype/feasibility_check/feasibility_check.js]: do not remove this

File: versa_system/versa_system/doctype/mockup_design/mockup_design.js [hunk 2]
```diff
@@ -1,8 +0,0 @@
-// Copyright (c) 2024, efeone and contributors
-// For license information, please see license.txt
-
-// frappe.ui.form.on("Mockup Design", {
-// 	refresh(frm) {
-
-// 	},
-// });
```
Comment (MhmdSinanKT) [inline versa_system/versa_system/doctype/mockup_design/mockup_design.js]: why was this deleted?

**Review by MhmdSinanKT — CHANGES_REQUESTED** [review]

File: versa_system/versa_system/doctype/mockup_design/mockup_design.js (hunk 2 above)
Comment (Sreelakshmi1831) [reply versa_system/versa_system/doctype/mockup_design/mockup_design.js]: accidenty happened

File: versa_system/versa_system/doctype/feasibility_check/feasibility_check.js (hunk 1 above)
Comment (Sreelakshmi1831) [reply versa_system/versa_system/doctype/feasibility_check/feasibility_check.js]: Ok , Should I add this now

File: versa_system/versa_system/doctype/mockup_design/mockup_design.js (hunk 2 above)
Comment (MhmdSinanKT) [reply versa_system/versa_system/doctype/mockup_design/mockup_design.js]: sure, please fix it

File: versa_system/versa_system/doctype/feasibility_check/feasibility_check.js (hunk 1 above)
Comment (MhmdSinanKT) [reply versa_system/versa_system/doctype/feasibility_check/feasibility_check.js]: yes

Comment (MhmdSinanKT) [reply versa_system/versa_system/doctype/feasibility_check/feasibility_check.js]: still not fixed

**Review by MhmdSinanKT — CHANGES_REQUESTED** [review]

File: versa_system/versa_system/doctype/feasibility_check/feasibility_check.js (hunk 1 above)
Comment (Sreelakshmi1831) [reply versa_system/versa_system/doctype/feasibility_check/feasibility_check.js]: how can I fix it

**Comment by Sreelakshmi1831** [comment]
check it now

**Review by MhmdSinanKT — APPROVED** [review]
